app/amigo-share-request-card/amigo-share-request-card.component.html

```diff
@@ -0,0 +1,36 @@
+<ion-card *ngIf="amigoShareRequest">
+
+    <ion-item>
+
+        <ion-avatar item-left>
+            <ion-icon name="person"></ion-icon>
+        </ion-avatar>
+
+        <h2>{{amigoShareRequest.sharedAmigo.name}}</h2>
+
+    </ion-item>
+
+    <ion-card-content>
+        <p>
+            {{amigoShareRequest.sharedAmigo.name}}
+            would like to be your amigo!
+        </p>
+    </ion-card-content>
+
+    <ion-row>
+        <ion-col>
+
+            <button secondary clear small>
+                <ion-icon name="remove-circle"></ion-icon>
+                <div>Decline</div>
+            </button>
+
+            <button primary clear small>
+                <ion-icon name="add-circle"></ion-icon>
+                <div>Accept</div>
+            </button>
```

Comment:
These buttons don't actually do anything yet.

They will eventually set `accepted` to `true` or `false` and then call a `save` method in the `AmigoShareRequestService` that will POST it to the server.

Just getting the page layout, components, and views in place in this PR.